extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml

```diff
@@ -676,6 +676,23 @@
           <value>org.exoplatform.social</value>
         </value-param>
       </init-params>
+    </component-plugin>  
+  </external-component-plugins>
+  
+  <external-component-plugins>
```

Comment:
@thomasdelhomenie ModelOverriderUpdaterPlugin must be kept for direct migrations from Platform 4.0.7/4.1.0/4.1.1 to 4.3.